### DISCUSSION TITLE
Usage to illustrate the potential of a tool like this to prioritise investment in safe routes to school

### DISCUSSION THREAD

**Discussion by Robinlovelace** [dabreegster/routing-engines#50] [General]
Heads-up @dabreegster I've just had a play with the tool around a case study location in York where I'll be presenting on SchoolRoutes worth with Sam and Emma on Thursday. It's looking really good! Assuming there's a school where the marker is, it reveals really interesting things about the LTS levels leading to the school:

![image](https://github.com/Urban-Analytics-Technology-Platform/od2net/assets/1825120/62061d2a-862b-4d6f-99ea-f5624fa29a84)

Opening a discussion in case you or anyone else want to chime in with thoughts, how can I improve the live demo on the day? I'll say of course that  this isn't real OD data which is an obvious on. Really impressed with this in any case and looking forward to demoing it as a prototype of what our outputs could look like.

Source: https://urban-analytics-technology-platform.github.io/od2net//interactive.html#12.87/53.92914/-1.08215

Cc @stholder3 and @EmmaVinterATE in case you want to give it a spin, we have more than enough content for the hour but this will add a level of web interactivity that will go down well I think!

**Comment by dabreegster** [comment]
One tip is to lower the Max requests from 1000 to about 100 to make the line widths more varied. https://github.com/Urban-Analytics-Technology-Platform/od2net/blob/1e9247f8d8e97b8c149c9641fde9d0563f4162cb/viewer/src/Layers.svelte#L19 controls line width right now; it's quite simplistic and often not useful
